src/battle/partner/bombette.c

```diff
@@ -1054,15 +1054,20 @@ EvtScript N(bomb) = {
             EVT_CALL(SetGoalToTarget, ACTOR_PARTNER)
             EVT_CALL(AddGoalPos, ACTOR_PARTNER, -40, 0, 0)
     EVT_END_SWITCH
-    EVT_CALL(MoveBattleCamOver, 60)
+    EVT_CALL(MoveBattleCamOver, 60 * DT)
     EVT_CALL(GetMenuSelection, LVar0, LVar1, LVar2)
     EVT_SWITCH(LVar2)
         EVT_CASE_EQ(MOVE_BOMB)
-            EVT_CALL(action_command_bomb_start, 0, 57, 3, 0)
+            EVT_CALL(action_command_bomb_start, 0, 57 * DT, 3, 0)
         EVT_CASE_EQ(MOVE_POWER_BOMB)
-            EVT_CALL(action_command_bomb_start, 0, 72, 3, 1)
+#if VERSION_PAL
+            // 72 * DT = 60
+            EVT_CALL(action_command_bomb_start, 0, 59, 3, 1)
```

Comment:
<img width="457" alt="image" src="https://user-images.githubusercontent.com/737603/227147368-e03a7394-0a5e-44b2-ab44-71d790c1784c.png">

Reply:
No clue what to do with this one.

Reply:
@HailSanta

Reply:
I think where its ambiguous with nothing hinting at what the coefficient should be, we should use whichever pairs with 1, just as a convention. So this would be 73 * DT - 1.